### PR TITLE
feat(select, autocomplete): ability to override options width

### PR DESCRIPTION
**Pull request by evtkhvch**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

Closes: #2638

- add `NbSelectComponent.optionsWidth` property
- add `NbAutocompleteComponent.optionsWidth` property

### CODE REVIEW

File: src/framework/theme/components/select/select.component.ts [hunk 1]
```diff
@@ -750,6 +761,8 @@ export class NbSelectComponent implements OnChanges, AfterViewInit, AfterContent
 
   protected keyManager: NbFocusKeyManager<NbOptionComponent>;
 
+  private _optionsWidth: number | undefined;
```
Comment (yggg) [inline src/framework/theme/components/select/select.component.ts:764]: Move above to the get/set declarations

File: src/framework/theme/components/autocomplete/autocomplete.component.ts [hunk 2]
```diff
@@ -63,6 +63,8 @@ export class NbAutocompleteComponent<T> implements AfterContentInit, OnDestroy {
    */
   _overlayPosition: NbPosition = '' as NbPosition;
 
+  private _optionsWidth: number | undefined;
```
Comment (yggg) [inline src/framework/theme/components/autocomplete/autocomplete.component.ts:66]: Move to the get/set declaration

**Review by yggg — CHANGES_REQUESTED** [review]

File: src/framework/theme/components/select/select.component.ts (hunk 1 above)
Comment (evtkhvch) [reply src/framework/theme/components/select/select.component.ts]: done

File: src/framework/theme/components/autocomplete/autocomplete.component.ts (hunk 2 above)
Comment (evtkhvch) [reply src/framework/theme/components/autocomplete/autocomplete.component.ts]: done

**Review by yggg — APPROVED** [review]

**Pull request merged**